helm_deploy/court-case-service/templates/ingress.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
     # Secure the retry all dlqs endpoint from outside of the Kubernetes ingress
     nginx.ingress.kubernetes.io/server-snippet: |
       server_tokens off;
-      location ~* ^/(queue-admin/retry-all-dlqs|process-un-resulted-cases) {
+      location ~* ^/(queue-admin/retry-all-dlqs|process-un-resulted-cases|hearing/delete-duplicates) {
```

Comment:
strange that this does not have a leading slash but as long as it works then it is fine